lgc/patch/Gfx9ConfigBuilder.cpp

```diff
@@ -98,7 +98,9 @@ void ConfigBuilder::buildPalMetadata() {
 // Builds register configuration for graphics pipeline (VS-FS), or FS-only shader compilation
 void ConfigBuilder::buildPipelineVsFsRegConfig() {
   GfxIpVersion gfxIp = m_pipelineState->getTargetInfo().getGfxIpVersion();
-  assert(gfxIp.major <= 10);
+  const bool partPipeline = !m_pipelineState->isWholePipeline() && m_pipelineState->hasShaderStage(ShaderStageFragment);
+  assert(gfxIp.major <= 10 || !partPipeline); // Must be GFX10 or below, or part-pipeline compilation
```

Comment:
Either the comment or the assert is wrong now, the assert has an or **not** `partPipeline`, the comment says ‘or part-pipeline‘.

Reply:
I made a typo :)